xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-test/xwiki-platform-like-test-docker/pom.xml

```diff
@@ -45,6 +45,13 @@
       <version>${project.version}</version>
       <type>xar</type>
     </dependency>
+    <!-- Required to display the like pages user profile UIX. -->
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-user-profile-ui</artifactId>
+      <type>xar</type>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Shouldn't this dep be declared somewhere else in the like modules? why is it only needed for testing and not for runtime?

Reply:
The user profile is only need to add the UIXP the load the UIX I want  to test. Should the UIX modules declare an optional dependency to the corresponding UIXP modules?

Reply:
See https://github.com/xwiki/xwiki-platform/pull/1625#discussion_r625027599